tools/wheelmaker.py

```diff
@@ -167,10 +167,12 @@ def add_wheelfile(self):
             wheel_contents += "Tag: %s\n" % tag
         self.add_string(self.distinfo_path("WHEEL"), wheel_contents)
 
-    def add_metadata(self, metadata, description):
+    def add_metadata(self, metadata, description, version):
         """Write METADATA file to the distribution."""
         # https://www.python.org/dev/peps/pep-0566/
         # https://packaging.python.org/specifications/core-metadata/
+        metadata += "Version: " + version
+        metadata += "\n"
         metadata += "\n"
```

Comment:
```suggestion
        metadata += "\n\n"
```

Oops, if preferred...